openreview/conference/builder.py

```diff
@@ -1212,6 +1212,14 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
         invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self, options)
         invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options(), options['reviewers_name'])
 
+        contact_email = 'info@openreview.net'
```

Comment:
can you add contact_info as a parameter in this function?

The request form recruitment process function should try to get the contact info from the request note and pass it to this function. 

The conference object can be used without a request form note, so it is better to parametrize this value.